tests/integration_tests/data_sources/test_clickhouse_ds.py

```diff
@@ -11,20 +11,25 @@ def test_clickhouse_ds():
     PORT = 8123
 
     clickhouse_url = f'http://{HOST}:{PORT}'
-    requests.post(clickhouse_url,
-                  data='CREATE DATABASE IF NOT EXISTS test')
-    requests.post(clickhouse_url, data='DROP TABLE IF EXISTS test.mock')
-    requests.post(clickhouse_url, data="""CREATE TABLE test.mock(
-            col1 String
-            ,col2 Int64
-            ,col3 Array(UInt8)
-        ) ENGINE=Memory""")
-    requests.post(clickhouse_url,
-                  data="""INSERT INTO test.mock VALUES ('a',1,[1,2,3])""")
-    requests.post(clickhouse_url,
-                  data="""INSERT INTO test.mock VALUES ('b',2,[2,3,1])""")
-    requests.post(clickhouse_url,
-                  data="""INSERT INTO test.mock VALUES ('c',3,[3,1,2])""")
+    queries = [
+        'CREATE DATABASE IF NOT EXISTS test',
+        'DROP TABLE IF EXISTS test.mock',
+        '''
+            CREATE TABLE test.mock(
+                col1 String
+                ,col2 Int64
+                ,col3 Array(UInt8)
+            ) ENGINE = MergeTree()
```

Comment:
Hmh, not sure about removing the `Memory` engine tests... but then again, probably superflous, we can rely on the database to work internally + `MergeTree` is the most widely used. So I'll call this good, go ahead and merge